doc/sphinx/credits.rst

```diff
@@ -1,12 +1,5 @@
-.. include:: preamble.rst
-.. include:: replaces.rst
-
 .. _credits:
 
--------------------------------------------
-Credits
--------------------------------------------
```

Comment:
The Credits chapter must be a proper chapter, independent of the introduction (see https://coq.inria.fr/refman/credits.html), so why remove the title from here?

Reply:
Argh, that one is tricky.  Top-level titles have to be in the index file for LaTeX.  I'll look for a fix.

Reply:
I wrapped the credits file into a `credits-wrapper` file.  It's ugly, but I don't think it'll hurt maintainability.